src/Screens/GameplayScreen/GameplayScreen.css

```diff
@@ -1,29 +1,36 @@
 #gameplayScreen {
-  font-family: Helvetica;
+	font-family: Helvetica;
 }
 
 /*-------------------
   Button Containers
 -------------------*/
 
+#upperRightButtonsContainer {
+	position: absolute;
+	left: 78vw;
```

Comment:
please align to the right, symetric to `upperLeftButtonsContainer`
(right:20px)